chemprop/data/utils.py

```diff
@@ -8,22 +9,33 @@
 
 from rdkit import Chem
 import numpy as np
+import pandas as pd
 from tqdm import tqdm
 
 from .data import MoleculeDatapoint, MoleculeDataset, make_mols
 from .scaffold import log_scaffold_stats, scaffold_split
 from chemprop.args import PredictArgs, TrainArgs
-from chemprop.features import load_features, load_valid_atom_or_bond_features, is_mol
+from chemprop.features import load_features, load_valid_atom_or_bond_features, is_explicit_h, is_adding_hs, is_mol
+from chemprop.rdkit import make_mol
+
+# Increase maximum size of field in the csv processing
+csv.field_size_limit(sys.maxsize)
```

Comment:
Why is this coming up?

Reply:
Line 19: `make_mol` is used to create RDKit molecule object in Line 416.

Line 21-22: For atom- and bond-level training, users can use either csv or pickle as input file. When using csv as input, it might appear `_csv.Error: field larger than field limit (131072)`, which means the csv file might contain very huge fields. This makes sense because the arrays of atom- and bond-level targets might be very large. In order to solve this, the field limit is set to sys.maxsize.